mne/beamformer/tests/test_lcmv.py

```diff
@@ -83,7 +83,7 @@ def _get_data(tmin=-0.1, tmax=0.15, all_forward=True, epochs=True,
                                    eeg=0.1, proj=True)
     if data_cov:
         with warnings.catch_warnings(record=True):
-            data_cov = mne.compute_covariance(epochs, tmin=0.04, tmax=0.15)
+            data_cov = mne.compute_covariance(epochs, tmin=0.04, tmax=0.145)
```

Comment:
I think it ended up failing here https://github.com/mne-tools/mne-python/blob/master/mne/beamformer/tests/test_lcmv.py#L111